cmake/common/gtest.cmake

```diff
@@ -24,7 +24,7 @@ macro(add_gtest)
         set(test "${ARGV0}")
         set(command "${test}")
     endif()
-    set(multiValueArgs SOURCES ENVIRONMENTS DEPENDENCIES LABELS)
+    set(multiValueArgs SOURCES ENVIRONMENTS DEPENDENCIES LABELS IGNORE)
```

Comment:
Not sure this was intented for this PR. Please check.

Reply:
Sorry, I departed from master but it was not aligned with the remote master branch. I've rebased already.